apps/web/src/pages/checkout/[packSlug]/index.tsx

```diff
@@ -0,0 +1,158 @@
+import {
+  PackAuction,
+  PackStatus,
+  PackType,
+  PublishedPack,
+} from '@algomart/schemas'
+import { GetServerSideProps } from 'next'
+import useTranslation from 'next-translate/useTranslation'
+import { useEffect } from 'react'
+
+import { ApiClient } from '@/clients/api-client'
+import { Analytics } from '@/clients/firebase-analytics'
+import { usePaymentProvider } from '@/contexts/payment-context'
+import { Environment } from '@/environment'
+import DefaultLayout from '@/layouts/default-layout'
+import {
+  getAuthenticatedUser,
+  handleUnauthenticatedRedirect,
+} from '@/services/api/auth-service'
+import CheckoutTemplate from '@/templates/checkout-template'
+import { urls } from '@/utils/urls'
+
+export interface CheckoutPageProps {
+  auctionPackId: string | null
+  currentBid: number | null
+  release: PublishedPack
+}
+
+export default function Checkout({
+  auctionPackId,
+  currentBid,
+  release,
+}: CheckoutPageProps) {
+  const { t } = useTranslation()
+  const paymentProps = usePaymentProvider({
+    auctionPackId,
+    currentBid,
+    release,
+  })
+
+  useEffect(() => {
+    Analytics.instance.beginCheckout({
+      itemName: release.title,
+      value: currentBid ?? release.price,
+    })
+  }, [currentBid, release])
+
+  return (
+    <DefaultLayout
+      pageTitle={
+        release.type === PackType.Auction
+          ? t('common:pageTitles.Placing Bid', { name: release.title })
+          : t('common:pageTitles.Checking Out', { name: release.title })
+      }
+      panelPadding
+    >
+      <CheckoutTemplate {...paymentProps} />
+    </DefaultLayout>
+  )
+}
+
+export const getServerSideProps: GetServerSideProps<CheckoutPageProps> = async (
+  context
+) => {
+  // Verify authentication
+  const user = await getAuthenticatedUser(context)
+  if (!user) {
+    return handleUnauthenticatedRedirect(context.resolvedUrl)
+  }
+
+  if (!Environment.isWireEnabled && !Environment.isCryptoEnabled) {
+    return {
+      redirect: {
+        destination: urls.checkoutPackWithMethod
+          .replace(':packSlug', context?.params?.packSlug as string)
+          .replace(':method', 'card'),
+        permanent: false,
+      },
+    }
+  }
```

Comment:
Instead of displaying the methods screen, if the feature flags aren't enabled, direct the user to the `/card` endpoint where they can fill out cc info.